labs/SAW/ceilLog2/Makefile

```diff
@@ -0,0 +1,20 @@
+# Note: Makefile assumes user already started the SAW remote API
+# $ start-saw-remote-api
+
+# Variables to hold relative file paths
+SRC=src
+PROOF=proof
+SPECS=specs
+ARTIFACTS=artifacts
+
+# Note: -g flag compiles the code with debug symbols enabled.
+#       The flag lets us pass field names in addition to indices for structs.
+all: artifacts
+	clang -c -g -emit-llvm -o $(ARTIFACTS)/ceilLog2.bc $(SRC)/ceilLog2.c
+	python3 $(PROOF)/ceilLog2.py
+
+artifacts:
+	mkdir $(ARTIFACTS)

```

Comment:
Better to test if the directory exists before creating it.

`[ -d artifacts ] || mkdir -p artifacts`

Reply:
Fair point. I added the `-p` flag for Makefiles calling `mkdir`.

Reply:
Thanks!